3p/integration.js

```diff
@@ -337,7 +335,10 @@ function triggerResizeRequest(width, height) {
 }
 
 function triggerRenderStart() {
-  nonSensitiveDataPostMessage('render-start');
+  if (!window.context.isRenderStartSent) {
```

Comment:
That's not a very expensive operation, though. If they really call it twice, it's on them, but it won't affect core code.